src/main/java/io/tiledb/cloud/TileDBUtils.java

```diff
@@ -12,17 +13,30 @@
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.HashMap;
 
 public class TileDBUtils {
 
+    /**
+     * Serializes the arguments to a String
+     *
+     * @param arguments The input arguments in a HashMap
+     * @return The arguments in a String
+     */
+    public static String serializeArgs(HashMap<String, Object> arguments) {
+        if (arguments == null || arguments.isEmpty()) return "";
+        Gson gson = new Gson();
+        return gson.toJson(arguments);
+    }
+
     /**
      *
      * This method gets bytes and translated them to arrow valuevectors
      * @param bytes The input bytes
      * @return A pair that consists of an ArrayList of all valueVectors and the number of batches read.
      * @throws IOException
      */
-    static io.tiledb.java.api.Pair<ArrayList<ValueVector>, Integer> createValueVectors(byte[] bytes) throws IOException {
+    protected static Pair<ArrayList<ValueVector>, Integer> createValueVectors(byte[] bytes) throws IOException {
```

Comment:
Is there a reason for this change?

Reply:
Yes, if you mean the switch to "protected"  it's because the method should only be used by classes in the same package, there is no reason to expose it. 
If you mean the removal of "io.tiledb.java.api." it's because we don't need to have the TileDB-Java API as a dependency in this repo. This was a typo in the initial implementation.

Reply:
SGTM thanks